src/node-capnp/capnp.cc

```diff
@@ -927,8 +927,13 @@ v8::Handle<v8::Value> setNative(const v8::Arguments& args) {
 v8::Local<v8::Object> schemaToObject(capnp::ParsedSchema schema, CapnpContext& context,
                                      v8::Handle<v8::Value> wrappedContext) {
   auto result = context.wrapper.wrap(new capnp::Schema(schema));
+  auto proto = schema.getProto();
+
+  result->Set(v8::String::NewSymbol("_id"),
```

Comment:
The leading underscore could be mistaken for "private". Let's call this `typeId`.